SPECS/usrsctp/usrsctp.spec

```diff
@@ -0,0 +1,67 @@
+Summary:        A portable SCTP userland stack
+Name:           usrsctp
+Version:        0.9.5.0
```

Comment:
we dont have to checkin any rpm or code just these files ?

Reply:
Right. The "spec" file tells rpm-build everything that it needs to know to make the software and install it.